harvard-cranfield-university.csl

```diff
@@ -270,25 +257,35 @@
             </group>
             <text variable="archive"/>
             <text variable="archive_location"/>
-            <text macro="online"/>
             <text macro="publisher"/>
           </group>
         </else-if>
-        <else-if type="article-journal article-magazine article-newspaper broadcast interview manuscript map patent personal_communication song speech thesis webpage motion_picture" match="any">
-          <group suffix=".">
+        <else-if type="article-journal" match="any">
+          <group delimiter=", ">
+            <text macro="title" prefix="'" suffix="'"/>
```

Comment:
Are you running the latest Mendeley version? (http://www.mendeley.com/release-notes/v1_13_4/)

If the style is set to "en-GB", it should use `punctuation-in-quote="false"` and single quotes by default.